### ISSUE TITLE
Make scales of axes configurable per dataset

### ISSUE THREAD

**Issue by YuriGlickman** [policycompass/featureboard#35]

**Comment by mmilaprat** [comment]
this issue is related with #5 ?

can you provide me more information about it?

The idea is to add something like this in the modal window?

![04-03-2016 11-20-11](https://cloud.githubusercontent.com/assets/6602727/13524406/491e2d42-e1fb-11e5-93a5-131f6d5ec9cb.jpg)

**Comment by YuriGlickman** [comment]
I think it is better to keep only "*" and "/" operands in the drop down list and let user enter the number in a edit window and define a new unit for the result.

**Comment by mmilaprat** [comment]
ok

**Comment by mmilaprat** [comment]
something like this?

![04-03-2016 13-04-31](https://cloud.githubusercontent.com/assets/6602727/13526680/c0b3caac-e209-11e5-9972-073192db78b2.jpg)

**Comment by YuriGlickman** [comment]
yes, and also the unit. I think it is a good option to be able to adjust the unit.

**Comment by mmilaprat** [comment]
units? units are part of the datasets, we cannot know conversions between units

**Comment by YuriGlickman** [comment]
you do not need to know. Just provide a field directly after multiplyer where user can define it as a string 

If my original dataset was on kg and I applyied:

"/" "1000"   when I should be able to write somethere what the result is in "gramm"

**Comment by mmilaprat** [comment]
ok, it can be possible to do with as an open text input field

**Comment by cbotsikas** [comment]
If i'm not mistaken, you are talking about unit scaling and this is not obvious seeing the above mock-ups.
IMO, unit scaling is always a decimal multiplier (eg 1km \* 1000 = 1000m, 1m \* 0.001 = 0.001km)
I would also like to see a more explanatory text like:
Unit scaling: 1 Unit = [      ] [      ]
where [     ] is a textbox
for example:
Unit scaling: 1 Unit = [ 0.001 ] [ gram ]

**Comment by cbotsikas** [comment]
it can also write:
Unit scaling: 1 Dataset Unit = [ ] [ ]

someone can easily figure out the multiplier [googling for it](https://www.google.com/search?q=unit+converter)

**Comment by mmilaprat** [comment]
As we don't have an internal scale conversion table my idea is to do something like the mock-up where is user who configure the scale conversion.

![04-03-2016 13-45-48](https://cloud.githubusercontent.com/assets/6602727/13527645/e32bede2-e210-11e5-8329-e3e35480c24b.jpg)

**Comment by YuriGlickman** [comment]
Personally, I like it.

Aslo I can imagine that in some situations users would like to change just the lable of the unit. With the proposed interface they can set "1" as multiplier and define another label

**Comment by cbotsikas** [comment]
I just find the operation select box confusing.
If you always have the information about the dataset unit you can have a unit section like:

Dataset Units
Scaling: 1 < dataset_unit > = [ 1 ] plot units
Unit label: [ < dataset_unit > ]

**Comment by fabiankirstein** [comment]
This is a complicated topic and we should answer some questions about it:
- What is the exact use case here? When does an user needs this? 
  - In my opinion the proposed solution is too technical for the user. He should just choose an unit. This was originally the plan. 
  - For more complex operations the user should generate a new metric
- Which component is responsible for such operations?
  - We are putting more and more into the visualizations. This should be done on a lower level (Datasets, Metrics....)

We should discuss this in the next telco.
